### ISSUE TITLE
[DOCS] Add ALT text to banner

### ISSUE THREAD

**Issue by Pradumnasaraf** [Pradumnasaraf/Post-My-Message#9]
### Description

Add alt text to the banner. The text should be **"post my message banner"**

### Screenshots

<img width="858" alt="Screenshot 2023-02-10 at 12 45 26 PM" src="https://user-images.githubusercontent.com/51878265/218026973-022b6747-6e4b-4f2d-b938-c6ba68c1aff1.png">

**Comment by Abhishek7Tech** [comment]
@Pradumnasaraf Hi I would like to contribute!

**Comment by Pradumnasaraf** [comment]
> @Pradumnasaraf Hi I would like to contribute!

Already assigned you one issue. So this will open for someone else

**Comment by AryanGupta99** [comment]
Hey @Pradumnasaraf  I'll do this one

**Comment by Pradumnasaraf** [comment]
Go ahead, Aryan
